how-not-to-be-a-dinosaur.md

```diff
@@ -0,0 +1,117 @@
+# How to _not_ be a dinosaur. And not give a fuck.
+
+This is my story about how I became an experienced senior frontend developer and how I had to start all over again. How it feltlike I had to start all over. How to learn. 
+
+This story is mostly about React, but might apply to other languages as well. Heck, it might even apply to completely different areas of expertise.
+
+## What happened? 
+
+It is 2019 now and I am 37 years old. I have been working for 15+ years professionally as a webdeveloper and -designer. After highschool I went to design school and learned to be a designer. But I have also always been interested in coding. In the early 90's I was already writing code in [GW-Basic](https://nl.wikipedia.org/wiki/GW-BASIC). I can recall writing my first lines of code in the last grade of primary school. Later I learned HTML and even a little PHP 3. In design school I also learned Flash and OOP in [Lingo](https://en.wikipedia.org/wiki/Lingo_(programming_language)). When my class got the task to code a simple dice rolling program, I wrote the complete Monopoly game. Just because writing code was a never ending path of discovery and wonder. And [a dose of dopamine](https://www.npr.org/sections/ed/2014/10/24/357811146/curiosity-it-may-have-killed-the-cat-but-it-helps-us-learn?t=1572532599814).
```

Comment:
```suggestion
It is 2019 now and I am 37 years old. I have been working for 15+ years professionally as a web developer and web designer. After high school, I went to design school and learned to be a designer. But I have also always been interested in coding. In the early 90's I was already writing code in [GW-Basic](https://nl.wikipedia.org/wiki/GW-BASIC). I can recall writing my first lines of code in the last grade of primary school. Later I learned HTML and even a little PHP 3. In design school, I also learned Flash and OOP in [Lingo](https://en.wikipedia.org/wiki/Lingo_(programming_language)). When my class got the task to code a simple dice-rolling program, I wrote the complete Monopoly game. Just because writing code was a never-ending path of discovery and wonder. And [a dose of dopamine](https://www.npr.org/sections/ed/2014/10/24/357811146/curiosity-it-may-have-killed-the-cat-but-it-helps-us-learn?t=1572532599814).
```